### DISCUSSION TITLE
Adding aria-hidden parameter

### DISCUSSION THREAD

**Discussion by wilson639** [spatie/statamic-responsive-images#185] [Q&A]
Basically, I want screen readers to ignore certain images on my site. Currently, I'm passing an image through to a html partial with:

`image: reponsive(image)`

and rendering with:

` {{ image | raw }}`

Which is why I believe I am unable to add HTML attributes as [described in the docs](https://statamic.com/addons/spatie/responsive-images)  (under "HTML Attributes").

`{{ responsive:image_field alt="" class="my-class" }}`

Ideally, I want to also pass through `hidden=true` which then appends the `aria-hidden=true` parameter onto the html tag. Is this possible with the way my project is setup?

Alternatively, is there a way for images without an alt text in the admin to not default back to the file name?

**Comment by ncla** [comment]
> Basically, I want screen readers to ignore certain images on my site. Currently, I'm passing an image through to a html partial with:
> 
> `image: reponsive(image)`

Where exactly does this code snippet appear? Is this Blade/Laravel controllers?

**Comment by wilson639** [comment]
This is inside of the .twig view file, I am using fractal and rendering an html partial and passing that responsive(image) through as a parameter.
________________________________

**Comment by ncla** [comment]
I have never used Twig/Fractcal, so it is unclear to me how are you initiating Responsive tag. However:

You could try calling the responsive tag through Statamic::tag with respective parameters you need https://statamic.dev/blade#using-tags-with-blade

You can also manually initiate Responsive tag, just have to get it right with params https://github.com/spatie/statamic-responsive-images/blob/main/src/Tags/ResponsiveTag.php

**Comment by wilson639** [comment]
Thank you for the help, I am new to statamic but that sounds promising. I will get back to you after I am able to test.
________________________________

**Comment by wilson639** [comment]
Hello, sorry for the late response. Unfortunately, it looks like this would require a lot more time than we expected due to how the project was originally setup.
Is it possible to allow blank alt tags ‘alt=“”’ if there is no title provided in the statamic admin? Currently it falls back to the file name. Thanks.

________________________________

**Reply by ncla** [reply]
You could maybe publish the blade template to your project and update it to your needs. Maybe that works.

https://github.com/spatie/statamic-responsive-images/blob/main/resources/views/responsiveImage.blade.php#L50-L52
